Desktop_Interface/isobuffer.h

```diff
@@ -24,80 +25,89 @@ enum class UartParity : uint8_t;
 // Imagine it as a circular buffer, but with access functions specifically
 // designed for isochronous data from an Xmega.
 
-#define CONSOLE_UPDATE_TIMER_PERIOD (ISO_PACKETS_PER_CTX * 4)
+constexpr auto CONSOLE_UPDATE_TIMER_PERIOD = ISO_PACKETS_PER_CTX * 4;
```

Comment:
Since this is used only in isoBuffer.cpp, it probably belongs in that namespace.
Also, it scares me a bit putting "auto" on an integral type.  Probably much more predicatable if we just used `uint32_t`